src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,99 @@
+package edu.kis.vh.nursery;
+
+import edu.kis.vh.nursery.list.IntArrayStack;
+import edu.kis.vh.nursery.list.IntLinkedList;
+import edu.kis.vh.nursery.list.ListInterface;
+
+public class DefaultCountingOutRhymer 
+// TODO what this does actually?
+{
+	public void push(int in) {
+		NUMBERS.push(in);
+	}
+
+
+	public boolean isEmpty() {
+		return NUMBERS.isEmpty();
+	}
+
+
+	public int top() {
+		return NUMBERS.top();
+	}
+
+
+	public int pop() {
+		return NUMBERS.pop();
+	}
+
+
+	private static final int MINUSE_ONE = 0;
+	// private final IntLinkedList NUMBERS = new IntLinkedList();
+	private final ListInterface NUMBERS;// = new IntArrayStack();
```

Comment:
This object is not a constant (although final). It is mutable. Correct name: numbers